test/node-disk-fill-test.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env mocha */
+
+'use strict';
+
+const os = require('os');
+const assert = require('bsert');
+const FullNode = require('../lib/node/fullnode');
+const RAMDisk = require('./util/ramdisk');
```

Comment:
Instead of using RAMDisk and making this test harder to run, we could just hijack each and every method that can fail for the expected errors. Thats more work, but we can start with simple chaindb.current.write monkeypatch.